.github/workflows/main.yml

```diff
@@ -55,15 +55,35 @@ jobs:
       - name: Install Carrier
         run: |
           ./cli/dist/carrier-linux-amd64 install -n
-      # - name: Push an App
+      - name: Carrier Setup Information
+        run: |
+          ./cli/dist/carrier-linux-amd64 info
+      - name: Default Organizations
+        run: |
+          ./cli/dist/carrier-linux-amd64 orgs
+      - name: Create Organization
+        run: |
+          ./cli/dist/carrier-linux-amd64 create-org myself
+      - name: Carrier Orgs
+        run: |
+          ./cli/dist/carrier-linux-amd64 orgs
+      - name: Push an App
+        run: |
+          sleep 8m ;# Wait for tekton to be usable
+          ./cli/dist/carrier-linux-amd64 create-org workspace ;# FUTURE: Part of install.
+          git config --global user.email "ci@suse.dev"
```

Comment:
I guess this is a fix for this: https://github.com/SUSE/carrier/issues/12
I'm currently fixing that issue. If this is merged first I will remove these 2 lines so that we check if my fix works.
If this is merged last, we should remove these lines on this PR.

Reply:
https://github.com/SUSE/carrier/pull/58

Reply:
This is now merged.

Reply:
So I can delete these lines I guess.

Reply:
https://github.com/SUSE/carrier/pull/60